components/dev-tools/easybuild/SPECS/easybuild.spec

```
@@ -16,16 +16,16 @@
 Summary:   Software build and installation framework
 Name:      EasyBuild%{PROJ_DELIM}
 Version:   4.6.2
-Release:   1%{?dist}
+Release:   %{?dist}.1
```

```suggestion
Release:   1%{?dist}.1
```
Maybe it should be? `EasyBuild-ohpc-4.6.2-1.ohpc.2.0.0.1.aarch64.rpm`, otherwise `EasyBuild-ohpc-4.6.2-.ohpc.2.0.0.1.aarch64.rpm`

---

The reason for the change to `%{?dist}.1` is that we can easily override it from our CI rebuild script.

OBS replaces the `Release:` any way during builds and in `OHPC_macros` it should be defined in such a way to make sense.

---

Oh, I noticed it:
https://github.com/openhpc/ohpc/blob/648a0deac6223c418e596e6c0800350d462e82ca/components/OHPC_macros#L31

`EasyBuild-ohpc-4.6.2-1.ohpc.2.0.0.0.aarch64.rpm` (2.6) to `EasyBuild-ohpc-4.6.2-27.ohpc.1.aarch64.rpm` (2.7)

So do we have plan to change this for each changed spec in future?